Standards/scs-XXXX-vN-Availability-Zones-Standard.md

```diff
@@ -0,0 +1,174 @@
+CSPs may need to communicate clearly up to which failure safety level their storage service can automatically have redundancy and from which level customers are responsible for the redundancy of their data.
+
+#### AZs in Network
+
+Network resources can be typically fastly and easily set up from building instruction.
+Those instructions are stored in the database of the networking service.
+
+If a physical machine, on which certain network resources are set up, is not available anymore, the resources can be rolled out on another physical machine, without being depended on the current situation of the lost resources.
+There might only be a loss of a few packages within the los network ressources.
+
+With having Compute and Storage in a good state (e.g. through having fire zones with a compute AZ each and storage being replicated over the fire zones) it would not have downsides to not have Availability Zones for the network service.
+It might even be the opposite: Having resources running in certain Availability Zones might permit them from being scheduled in other AZs[^3].
+This standard will therefore make no recommendations about Network AZs.
+
+[^3]: [Availability Zones in Neutron for OVN](https://docs.openstack.org/neutron/latest/admin/ovn/availability_zones.html)
+
+### Open questions
+
+Without the networking AZs we only need to take a closer look into attaching volumes to virtual machines across AZs.
+
+It is possible to allow or forbid cross-attaching volumes from one AZ to virtual machines in another AZ.
+If it is not allowed, then the creation of volume-based virtual machines will fail, in case of an outage of a complete Availability Zone.
+This does not seem to be a good option in regard for the failure safety level, as transfering a virtual machine from one AZ to another in a failure case will get way more complex.
+A replication of the volume has to be present in another storage Availability Zone that can be attached to the corresponding compute Availability Zone, which is not the AZ, that has an outage.
+Then this replication - maybe a snapshot - can be used to create a new virtual machine.
+
+While it seems to be a good decision to allow cross-attach, CSPs currently do not seem to widely use it.
+The reasons for and against this configuration may need to be discussed further to decide, whether this standard should make any recommendations regarding cross-attach.
+
+## Standard
+
+Compute Availability Zone MUST be in different fire zones.
+Availabilty Zones for Storage SHOULD be setup, if there is no storage backend used that can span over different fire zones and automatically replicate the data.
+
+[TO BE DISCUSSED:] If Availability Zones for Storage are used, the attaching of volumes from one Storage Availability Zone to another Compute Availability Zone (cross-attach) SHOULD be allowed.
+
+Within each Availability Zone:
+
+- there MUST be redundancy in power supply, as in line into the deployment
```

Comment:
This is an important topic. But I wonder, if this would fit in a standard about Availability Zones in that kind of detail.

I did not wanted to discuss all possible physical features, that need to be redundant. I think it would be better to refer to a document, doing all this (at least within this standard). Maybe something from the BSI like [this](https://www.bsi.bund.de/SharedDocs/Downloads/DE/BSI/Hochverfuegbarkeit/BandG/G7_HV-Prinzipien.pdf?__blob=publicationFile&v=3)?

Or do you think that it should be stated here in a detail like:
- there MUST be at least two redundant power sources (power line  or generator)
- each PDU SHOULD have at least one redundant twin
- each PDU MUST have at least two redundant electric circuits to the redundant power sources

Reply:
I think it should in the eyes of the CSP (also giving them certain interpretation freedom) while expecting that user is able to have certain level of high-er availability.

Reply:
I think the status quo with deferring to the BSI standards is fine, thus marking this as resolved.